www/src/pages/getting-started/introduction.mdx

```diff
@@ -38,11 +40,53 @@ more information about the benefits of using a CDN can be found
 
 <CssCodeBlock />
 
-For more advanced use cases you can also use a bundler like Webpack
-or Browserify to include the CSS files for you as part of your build
-process but that is beyond the scope of this guide. Also see
-[the Bootstrap docs](https://getbootstrap.com/docs/4.3/getting-started/theming/) 
-for details about customizing stylesheets to match your component
+### Sass
+
+In case you are using **Sass** the simplest way is to include the Bootstrap’s source Sass files
+in your main Sass file and then require it on your `src/index.js` or `App.js` file.
+This applies to a typical `create-react-app` application in other use cases you might have to setup
+the bundler of your choice to compile Sass/SCSS stylesheets to css.
```

Comment:
```suggestion
the bundler of your choice to compile Sass/SCSS stylesheets to CSS.
```

Reply:
I was just following what was already there:
```
...
Line 26 - Bootstrap, we don't ship with any included css. However, some
...
```
But I can change it, please confirm 😄

Reply:
I vote to change this to CSS to be correct. And change the other css to CSS as well IMO.